File: LeetCode_1523/Main.py
class Solution(object):
    def countOdds(self, low, high):
        """
        :type low: int
        :type high: int
        :rtype: int
        """
        def func(n):
            return (n + 1) // 2
        return func(high) - func(low - 1)

File: LeetCode_1523/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int func(int n) {
        return (n + 1) / 2;
    }
    int countOdds(int low, int high) {
        return func(high) - func(low - 1);
    }
};